context/build-utils/webpack.common.js

```diff
@@ -31,6 +31,7 @@ const config = {
         test: /\.js$/,
         use: ['source-map-loader'],
         enforce: 'pre',
+        exclude: [/node_modules\/lineupjs[x]?/],
```

Comment:
```suggestion
        exclude: [/node_modules\/lineupjsx?/],
```
Should do the same thing?